custom/icds/location_reassignment/parser.py

```diff
@@ -469,20 +470,36 @@ def __init__(self, domain, workbook):
 
     def parse(self):
         errors = []
+        site_codes = set([ws.title for ws in self.workbook.worksheets])
+        location_ids = {loc.site_code: loc.location_id
+                        for loc in SQLLocation.objects.filter(domain=self.domain, site_code__in=site_codes)}
         for worksheet in self.workbook.worksheets:
             location_site_code = worksheet.title
+            location_id = location_ids[location_site_code]
+            _, case_ids_expected = get_case_ids_for_reassignment(self.domain, location_id)
```

Comment:
the only difference from `HouseholdReassignmentParser` implementation above.
DRYed up in later commit.